test/unit/presenters/api/applications_new_presenter_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Api::ApplicationsNewPresenterTest < ActiveSupport::TestCase
+
+  Presenter = Api::ApplicationsNewPresenter
+
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+    @service = FactoryBot.create(:service)
+    @user = FactoryBot.create(:simple_admin)
+  end
+
+  attr_reader :provider, :service, :user, :cinstance
+
+  test 'new_application_form_data' do
+    presenter = Presenter.new(provider: provider, service: service, user: user)
+    form_data = presenter.new_application_form_data
+
+    assert form_data.key? :"create-application-path"
+    assert form_data.key? :"create-application-plan-path"
+    assert form_data.key? :"service-subscriptions-path"
+    assert form_data.key? :"service-plans-allowed"
+    assert form_data.key? :"defined-fields"
+    assert form_data.key? :"most-recently-created-buyers"
+    assert form_data.key? :"buyers-count"
+    assert form_data.key? :product
```

Comment:
I agree it is more verbose but you could do:

`assert_same_elements expected_keys, form_data.keys `